src/locales/en_US.ts

```diff
@@ -1139,6 +1139,8 @@ const strings = {
   user_closed_modal_no_wallet: 'User closed modal. No wallet was selected',
   plugin_powered_by_space: 'Powered by ',
   tap_to_change_provider: 'Tap to Change Provider',
+  tap_to_view_quote_amount_and_rate: 'Tap to view quote amounts and rates',
```

Comment:
Avoid "Tap to X." We don't want our app to be littered with "Tap to do this," "Tap to do that" - we might as well be a 90's web page with underlined blue links.

Ideally it's the UI/UX design's job to communicate tappability clearly, but suggest "Select to view quote" at as a slightly better copy